### ISSUE TITLE
combine CapillaryPressureVG and EffectiveSaturationVG

### ISSUE THREAD

**Issue by WilkAndy** [WilkAndy/moose#41]
Tagging @cpgr .  These are inverses of each other, so i think it would be nicer in one class.  Not urgent before the first PR to moose/devel.

a

**Comment by cpgr** [comment]
Yeah, I thought of doing it that way originally. Happy to put them together - I'll do it tomorrow. Shouldn't take long
